packages/app-store/zapier/lib/nodeScheduler.ts

```diff
@@ -57,38 +58,39 @@ export async function cancelScheduledJobs(
   appId?: string | null,
   isReschedule?: boolean
 ) {
-  try {
-    let scheduledJobs = booking.scheduledJobs || [];
+  let scheduledJobs = booking.scheduledJobs || [];
+  if (!booking.scheduledJobs) return;
```

Comment:
What's the point of fallback initialization with empty array if we are still early returning in that case.